Makefile

```diff
@@ -133,7 +133,7 @@ helm-uninstall:
 	kubectl delete -f https://raw.githubusercontent.com/kubesphere/ks-installer/master/roles/ks-core/prepare/files/ks-init/role-templates.yaml
 
 # Run tests
-test: vet
+test: vet fmt goimports
```

Comment:
I have a second thought, we use `make test` in Prow, adding `fmt goimports` will slow our pipeline. We have presubmit checks to find those unformatted PRs. For developers, `make verify-all` before submitting is a good practice.

Reply:
`make verify-all` is a definitely good solution to do the check work. But this is my first time to know that makefile command. It's quite important that letting contributors know fewer things before they get started their contributions. How about the following solution:

```
test: vet verify-all
  xxx

test-without-verify: vet
  xxx
```

then let Prow runs the command `make test-without-verify`.

Reply:
I think it's better to make it more clear that contributors should run `make verify-all` before submitting in our development guide. Also, if they don't run `make verify-all`, our presubmit will complain and they will do it next time :)

Reply:
Ok than. But I still believe that the process should be as easy as possible. I'm going to close this PR until others want to simplify the process.